chart-port.patch

```diff
@@ -157,6 +148,25 @@ index fcb57c0..779400c 100644
          # load the data
          for row in chart_data:
              self._add_value(
--- 
-2.19.1
-
+diff --git a/sugarpycha/chart.py b/sugarpycha/chart.py
+index 265eb63..2a17f89 100644
+--- a/sugarpycha/chart.py
++++ b/sugarpycha/chart.py
+@@ -140,7 +140,7 @@ class Chart(object):
+ 
+         # Remove invalid args before calling the constructor
+         kwargs = dict(self.options.colorScheme.args)
+-        validArgs = inspect.getargspec(colorSchemeClass.__init__)[0]
++        validArgs = inspect.getfullargspec(colorSchemeClass.__init__)[0]
+         kwargs = dict([(k, v)
+                       for k, v in list(kwargs.items()) if k in validArgs])
+         self.colorScheme = colorSchemeClass(keys, **kwargs)
+@@ -360,7 +360,7 @@ class Chart(object):
+ 
+     def _renderTick(self, cx, tick, x, y, x2, y2, rotate, text_position):
+         """Aux method for _renderXTick and _renderYTick"""
+-        if isinstance(tick, collections.Callable):
++        if isinstance(tick, collections.abc.Callable):
```

Comment:
I grep'd the code, if this is the fix, we should probably fix it everywhere else.

```
grep "collections.Callable" -r .
./chart/sugarpycha/bar.py:                if isinstance(self.options.yvals.renderer, collections.Callable):
./chart/sugarpycha/chart.py:        if isinstance(tick, collections.Callable):
./chart/sugarpycha/polygonal.py:        if isinstance(tick, collections.Callable):
./chart/sugarpycha/polygonal.py:        if isinstance(tick, collections.Callable):
./chart/sugarpycha/radial.py:        if isinstance(tick, collections.Callable):
./chart/sugarpycha/radial.py:        if isinstance(tick, collections.Callable):

```